shared/src/models/patient.ts

```diff
@@ -131,8 +131,21 @@ export class Patient {
     @Min(0)
     public readonly timeSpeed: number = 1;
 
+    @IsNumber()
+    @Min(0)
+    public treatmentTime = 0;
+
     static readonly create = getCreate(this);
 
+    static readonly msUntilRealStatus: number = 120_000;
```

Comment:
```suggestion
    static readonly msUntilRealStatus: number = 2 * 60 * 1000;
```

This makes it easier to see that this are two minutes.